advanced/Scripts/list.sh

```diff
@@ -215,8 +215,8 @@ for var in "$@"; do
     case "${var}" in
         "-w" | "whitelist"   ) listType="whitelist"; listAlt="blacklist";;
         "-b" | "blacklist"   ) listType="blacklist"; listAlt="whitelist";;
-        "--wild" | "wildcard" ) listType="regex"; wildcard=true;;
-        "--regex" | "regex"   ) listType="regex";;
+        "--wild" | "wildcard" ) listType="regex_blacklist"; wildcard=true;;
+        "--regex" | "regex"   ) listType="regex_blacklist";;
```

Comment:
Is the plan to only support regex whitelist when we integrate the new web interface and API?

Reply:
They are still to be added. This reminds me of why this PR is still marked as "in progress".

According adaptions to the current web interface will have to be made as well. However, I can only fully address this next week as I already exceeded my mobile data limit for this month.

As for the other tables, the implementation will be very basic (addition and removal of the filters). The API can then introduce more powerful features such as editing the comments, enabling/disabling, group management, etc.